posthog/api/capture.py

```diff
@@ -98,11 +105,11 @@ def _get_token(data, request) -> Optional[str]:
     return None
 
 
-# Support test_[apiKey] for users with multiple environments
-def _clean_token(token):
-    is_test_environment = token.startswith("test_")
-    token = token[5:] if is_test_environment else token
-    return token, is_test_environment
+def get_token(data, request) -> Tuple[Optional[str], bool]:
```

Comment:
It's sort of weird we have both `_get_token` and `get_token` functions, I can see myself confused. How about refactoring the existing `_get_token` function?